frontend/lib/src/components/widgets/NumberInput/NumberInput.test.tsx

```diff
@@ -295,73 +275,56 @@ describe("NumberInput widget", () => {
       )
     })
 
-    it("calls onChange", () => {
-      const props = getIntProps({ default: 10 })
-      const wrapper = shallow(<NumberInput {...props} />)
-
-      const InputWrapper = wrapper.find(UIInput)
+    it("calls onChange", async () => {
```

Comment:
can we just change the title of this since we are not checking whether or not we are calling onChange?